stdlib/2.7/getopt.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import List, Tuple
+
+class GetoptError(Exception):
+    opt = ... # type: str
+    msg = ... # type: str
+    def __init__(self, msg: str, opt: str=...) -> None: ...
+    def __str__(self) -> str: ...
+
+error = GetoptError
+
+def getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str, str]],
+                                             List[str]]: ...
+
+def gnu_getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str,str]],
+                                             List[str]]: ...
+
+def do_longs(opts: List[Tuple[str,str]], opt: str,
```

Comment:
I am not trying to insult it, but compared to e.g. the Java documentation the Python documentation is lacking a lot. Often it does not even describe all the parameters and the programmer is left to figure that stuff out for themselves. Preconditions and postconditions are also rarely described.

But that's a bit offtopic. I will remove the undocumented functions.